### PR TITLE
10449: Notice IVLR 8 (UQHP to UQHP Final Renewal Notice) 

### PR DESCRIPTION
**Pull request by davidplappert**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [10449](https://devops.dchbx.org/redmine/issues/10449) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | n/a |
| Development Story Points | 1 |
| Peer Review Story Points | n/a |
| Ticket Author | Camille Gray |

### CODE REVIEW

File: app/notices/ivl_notices/variable_ivl_renewal_notice.rb [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    args[:notice] = PdfTemplates::ConditionalEligibilityNotice.new
+    args[:market_kind] = 'individual'
+    args[:recipient_document_store]= consumer_role.person
+    args[:to] = consumer_role.person.work_email_or_best
+    self.data = args[:data]
+    self.header = "notices/shared/header_with_page_numbers.html.erb"
+    super(args)
+  end
+
+  def deliver
```
Comment (Battula) [inline app/notices/ivl_notices/variable_ivl_renewal_notice.rb]: I don't think you need this method here

Comment (blackeaglejs) [inline app/notices/ivl_notices/variable_ivl_renewal_notice.rb]: @Battula do you mean the deliver method? I'm using that to make sure I get the right groups of pages...

File: app/notices/ivl_notices/variable_ivl_renewal_notice.rb [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    notice.primary_fullname = recipient.full_name.titleize || ""
+    if recipient.mailing_address
+      append_address(recipient.mailing_address)
+    else  
+      # @notice.primary_address = nil
+      raise 'mailing address not present' 
+    end
+  end
+
+  def append_data
```
Comment (blackeaglejs) [inline app/notices/ivl_notices/variable_ivl_renewal_notice.rb]: Yeah, the subscriber data.

Comment (Battula) [inline app/notices/ivl_notices/variable_ivl_renewal_notice.rb]: I don't see any of this data being used in the notice template

**Pull request merged**